### ISSUE TITLE
Using the keybind for "Open without auto hide" makes my character roll

### ISSUE THREAD

**Issue by Kesacoli** [Kvan7/Exiled-Exchange-2#85]
***Describe what you are experiencing (or other stuff)***

I rebinded the keybind "open without autohide" to "Numpad 0" and even though my dodge roll is on "Alt", it makes my character compulsively roll. I do have a french keyboard configured in AZERTY but my game is in english.
I tried remapping it to say "Shift + A", "Shift + Q", "Shift + CTRL + Q" and it keeps making my character roll when pressed for some reason...

Edit : Scratch that, it does it on other keybinds as well (for all the keybinds in the "hotkeys" section, but not for the binds in the "chat" section

***Steps to reproduce, if needed***

Change keybind for a hotkey in the "hotkeys" section and press the button, even when not hovering an item or in inventory it should make the character roll

**Comment by Kvan7** [comment]
If possible can you check if this is a bug in the original awakened? if it isn't something i broke, i am probably going to leave it for now while I'm still getting EE2 generally working for poe2. Sorry if this is a deal breaker for you, I would at this time rather get the program fully working over upstream bugs

**Comment by Kesacoli** [comment]
I never used Awakened Poe Trade considering I barely played the first game, so unfortunately I can't reproduce the bug in POE 1.

It is rather debilitating however considering you can't use the keybind when viewing your stash or talking to a vendor (because it autocloses when you roll), but you can circumvent that by simply using the CTRL + C / CTRL + V method, or through your inventory, so the program is still usable. albeit a bit janky

I think you're right though, focusing on the bigger core of the program is probably better rather than bothering with QOL inconveniences.

**Comment by Kvan7** [comment]
If you can install awakened, i can provide you with a config file to get it to open in poe2 so we can see which it is, if you would like

**Comment by Kesacoli** [comment]
aight, I just installed awakned, give me the config file pls, also where do I put it ?

**Comment by Kvan7** [comment]
Run awakened, kill it with task manager (to generate required folders)

Go to `%APP_DATA%/awakened-poe-trade`

put this config.json in `apt-data` overwrite if something is already there
[config.json](https://github.com/user-attachments/files/18172406/config.json)

**Comment by Kvan7** [comment]
Should be able to then run awakened poe trade (make sure EE2 is not running) and launch poe2 to use it



> aight, I just installed awakned, give me the config file pls, also where do I put it ?
> 
> edit : do I just replace the config.json ? In "AppData\Roaming\awakened-poe-trade\apt-data"

yeah or just copy over the config, that _should_ work too

**Comment by Kesacoli** [comment]
Everything seems to be working but I can't open the overlay  in game by using "Shift + Space", it just doesn't do anything

**Comment by Kvan7** [comment]
hmm well that test didn't do anything then. feel free to uninstall awakened then since that didn't help to solve the problem.

> I rebinded the keybind "open without autohide" to "Numpad 0" and even though my dodge roll is on "Alt", it makes my character compulsively roll. I do have a french keyboard configured in AZERTY but my game is in english.

Ahh, i think i know. "open without autohide" is just the price check. This presses `ctrl+alt+c` to copy the text of the items, so it is pressing alt and rolling you. You can _try_ rebinding in settings>input "show advanced item descriptions" to a different key, restarting both poe2 and ee2 and see if that works. I think this reads the ini file so should pick up on the change in keybindings but I'm not certain.

**Comment by Kesacoli** [comment]
problem is that my keybind for "show advanced item descriptions" is already another bind, it's "shift" for me

![image](https://github.com/user-attachments/assets/51beb4e5-16d3-4337-852b-8d48c92ec79f)

Or do you mean rebinding dodging to another key than "alt" ?

**Comment by Kvan7** [comment]
yeah that's what i meant. I'll need to double check the config parser to see if the format changed and I need to update it.

If it is that, then this will get added to backlog and I'll get to it soon. if it isn't in the end that, then this will have to be tracked down and will take a bit more time. Sorry for the inconvenience this is causing.

**Comment by Kesacoli** [comment]
No worries, thanks for taking the time to try to fix !
